stac_fastapi/extensions/setup.py

```diff
@@ -0,0 +1,56 @@
+"""arturo-stac-api."""
+import os
+from imp import load_source
+
+from setuptools import find_namespace_packages, setup
+
+with open("README.md") as f:
+    desc = f.read()
+
+# Get version from stac-fastapi-api
+__version__ = load_source(
+    "stac_fastapi.extensions.version",
+    os.path.join(os.path.dirname(__file__), "stac_fastapi/extensions/version.py"),
+).__version__  # type:ignore
+
+install_requires = [
+    "fastapi",
+    "attrs",
+    "pydantic[dotenv]",
+    "stac_pydantic==1.3.8",
+    "stac-fastapi.types",
+    "stac-fastapi.api",
```

Comment:
In the future we might want to do

```
f"stac-fastapi.types=={__version__}"
```

if we think it's important users have the same versions installed.